docsource/buildandconfigure.rst

```diff
@@ -3,79 +3,30 @@
-scripts. These instructions are for deploying the server and so do not
-address the scripts. Building outside the ``dev_container`` means the Makefile
-uses several default values for deployment - if you wish to use other values
-deploy from the ``dev_container`` as usual.
-
 Build the workspace service
 ---------------------------
 
-First checkout the ``dev_container``::
-
-    /kb$ sudo git clone https://github.com/kbase/dev_container
-    Cloning into 'dev_container'...
-    remote: Counting objects: 1097, done.
-    remote: Total 1097 (delta 0), reused 0 (delta 0), pack-reused 1097
-    Receiving objects: 100% (1097/1097), 138.81 KiB, done.
-    Resolving deltas: 100% (661/661), done.
-
-.. note::
-   In the v26 image, ``/kb`` is owned by ``root``. As an alternative to
-   repetitive ``sudo`` s, ``chown`` ``/kb`` to the user.
-
-Bootstrap and source the user environment file, which sets up Java and Perl
-paths which the WSS build needs::
-
-    /kb$ cd dev_container/
-    /kb/dev_container$ sudo ./bootstrap /kb/runtime/
-    /kb/dev_container$ source user-env.sh
-
-Now the WSS may be built. If building inside the ``dev_container`` all the
-dependencies from the ``DEPENDENCIES`` file are required, but to build outside
-the ``dev_container``, only the ``jars`` and ``workspace_deluxe`` repos are
-necessary::
+Get the code::
 
     ~$ mkdir kb
     ~$ cd kb
     ~/kb$ git clone https://github.com/kbase/workspace_deluxe
-    Cloning into 'workspace_deluxe'...
-    remote: Counting objects: 21961, done.
-    remote: Compressing objects: 100% (40/40), done.
-    remote: Total 21961 (delta 20), reused 0 (delta 0), pack-reused 21921
-    Receiving objects: 100% (21961/21961), 21.42 MiB | 16.27 MiB/s, done.
-    Resolving deltas: 100% (13979/13979), done.
-
-    ~/kb$ git clone https://github.com/kbase/jars
-    Cloning into 'jars'...
-    remote: Counting objects: 1466, done.
-    remote: Total 1466 (delta 0), reused 0 (delta 0), pack-reused 1466
-    Receiving objects: 100% (1466/1466), 59.43 MiB | 21.49 MiB/s, done.
-    Resolving deltas: 100% (626/626), done.
+
+Build::
 
     ~/kb$ cd workspace_deluxe/
-    ~/kb/workspace_deluxe$ make
+    ~/kb/workspace_deluxe$ ./gradlew buildAll
     *snip*
 
-``make`` will build:
-
-* A workspace client jar in ``/dist/client``
-* A workspace server jar in ``/dist``
-* This documentation in ``/docs``
+``buildAll`` will build 3 jars in ``build/libs``:
 
-.. note::
-   If the build fails due to a sphinx error, sphinx may require an upgrade to
-   >= 1.3::
+* A workspace client jar
+* A workspace server WAR file
+* A workspace shadow jar containing all test code. This is useful for starting a workpace server
+  from other processes without needing a docker container, but should **only** be used for testing.
 
-       $ sudo pip install sphinx --upgrade
+It will also build the ``build\update_workspace_database_schema`` script which is used to
```

Comment:
is that the right direction for the slash?

Reply:
Nope, fixed. No DOS here please